lib/assert/imgLoaded.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var assert = require('assertive');
+var imgLoadedFn = require('./imgLoaded_client');
+var _ = require('lodash');
+
+exports.imgLoaded = function imgLoaded(doc, selector) {
+  if (arguments.length === 1) {
+    selector = doc;
+    doc = undefined;
+  }
+  assert.hasType('imgLoaded(selector) - requires (String) selector', String, selector);
+
+  var result = this.browser.evaluate(selector, imgLoadedFn);
+  if (result === true) {
+    return;
+  }
+
+  function fail(help) {
+    var message = 'imgLoaded ' + JSON.stringify(selector) + ': ' + help;
```

Comment:
I could replace it with `util.format('imgLoaded %j: %s', selector, help)`. But I was too lazy for that. ^^